.travis.yml

```diff
@@ -5,15 +5,26 @@ env:
 install:
   - pip install GitPython
   - export TRAVIS_COMMIT_MSG="$TRAVIS_REPO_SLUG - $(git log --format=%B --no-merges -n 1)"
-  - git clone https://github.com/xbmc-catchuptv-au/repo-devel.git .deploy
 script:
-  - cd .deploy
-  - ./manage_repo.py ../
+  - 'if [ -z "$TRAVIS_TAG" ]; then
+         git clone https://github.com/glennguy/repo-devel.git .deploy || travis_terminate 1;
+         cd .deploy || travis_terminate 1;
+         ./manage_repo.py ../ || travis_terminate 1;
```

Comment:
For some of these, I'd be fine with 
git clone https://github.com/xbmc-catchuptv-au/repo-devel.git .deploy && cd .deploy && ./manage_repo.py ../
all on one line maybe?

Also, what is the benefit of the travis_terminate?

Reply:
I read on a blog somewhere that when using if statements Travis won't know if any errors occurred in the block, the if always returns 0. This way makes sure errors get passed back. Haven't actually tested it myself, I was just trying to cover all bases.